ppb/vfs.py

```diff
@@ -0,0 +1,97 @@
+"""
+Handles opening files from the Python "VFS".
+"""
+import logging
+from pathlib import Path
+import sys
+try:
+    import importlib.resources as impres
+except ImportError:
+    # Backport
+    import importlib_resources as impres
+
+logger = logging.getLogger(__name__)
+
+
+def _main_path():
+    main = sys.modules['__main__']
+    mainpath = getattr(main, '__file__')
+    if mainpath:
+        mainpath = Path(mainpath)
+        return mainpath.resolve().parent
+    else:
+        return Path.cwd()
```

Comment:
Document when `Path.cwd()` is expected to be used, and why it's a good choice.

Reply:
This is loosely based on the deleted `__resource_path__` implementation, where under a REPL `__main__` will refer to the console namespace and not have a `__file__`. (It used [`getfile`](https://github.com/python/cpython/blob/3.7/Lib/inspect.py) though)